### ISSUE TITLE
I can't choose what distribution channel (stable, beta or nightly) of Brave I want to use

### ISSUE THREAD

**Issue by ceskyDJ** [ImranR98/Obtainium#1449]
**Describe the bug**

I want to manage installation and update processes Brave via Obtainium. Brave has more distribution channels — Stable (release with latest tag and Release in its name), Beta (release with Beta in its name), Nightly (release with Nightly in its name) and pre-release versions of these channels. Obtainium hasn't support for complete resolution of more distribution channels (it's maybe a candidate for a feature request), but has an ability to require latest tag. So, it should support dividing of Stable and other distribution channels, which is enough for me. When I setup Obtainium to require latest tag ("Verify the 'latest' tag" option), it ignores it and downloads newest Nightly/Beta build, which doesn't have the latest tag. It even shows version without latest tag (Nightly or Beta) as the newest one. So, obviously the verification of the latest tag doesn't work.

**To Reproduce**

I installed Brave using Obtainium a few days ago, so my current version is the last Nightly version (Obtainium should use last Stable version but used this one). Now, it says the latest version is Beta (without latest tag):
![Screenshot_20240305_145401](https://github.com/ImranR98/Obtainium/assets/48548230/5f40c6b2-0bbe-4daf-8186-4b48b394f6b7)

**Please complete the following information:**
 - Device: Samsung Galaxy S22
 - OS: Android 14
 - Obtainium Version: v1.0.4

**Comment by ceskyDJ** [comment]
For completeness, these are current latest versions of each distribution channel:
- **Stable:** https://github.com/brave/brave-browser/releases/tag/v1.63.165
- **Beta:** https://github.com/brave/brave-browser/releases/tag/v1.64.95
- **Nightly:** https://github.com/brave/brave-browser/releases/tag/v1.65.53

**Comment by DwainZwerg** [comment]
Can you check whether the “Include prereleases” toggle is deactivated and the “Fallback to older releases” toggle is activated? If not, can you try whether the problem then also occurs?

**Comment by ceskyDJ** [comment]
> Can you check whether the “Include prereleases” toggle is deactivated and the “Fallback to older releases” toggle is activated?

Yes, of course. **"Include preleases"** is set to `false` and **"Fallback to older releases** is set to `true`.

**Comment by ImranR98** [comment]
Found the bug in the 'verify latest' setting. Will be fixed in the next release.

**Comment by ImranR98** [comment]
We already have a release title filter (remember to turn on 'include prereleases' too).

**Comment by ceskyDJ** [comment]
Oh, I see. I'm sorry I'm totally blind. Thanks for the information.

**Comment by ImranR98** [comment]
https://github.com/ImranR98/Obtainium/releases/tag/v1.0.5
